### DISCUSSION TITLE
Alinhamento da po-page-dynamic-table dentro de um po-page-default

### DISCUSSION THREAD

**Discussion by gtrabach** [po-ui/po-angular#1767] [Tire suas dúvidas] [answered]
Estou com problemas ao colocar um componente do tipo po-page-dynamic-table dentro de um po-page-default.

Como o po-page-default já cria um espaçamento para que o menu não o encubra dentro da página, ao colocar um po-page-dynamic-table dentro dele, é criado outro espaçamento, quebrando o layout.

Como posso fazer para corrigir este problema? No caso, remover o espaçamento adicionado pelo po-page-dynamic-table.

Observação: Para simular, a tela precisa estar do tamanho onde o menu caiba.

Exemplo: https://po-ui-st8kou.stackblitz.io

![image](https://github.com/po-ui/po-angular/assets/46979335/75818370-275a-462c-a364-3cc50e246aa8)

**Comment by anliben** [comment]
Bom Dia @gtrabach, tudo bem?

Recebemos sua issue e em breve iremos analisar, porem poderia montar um exemplo no stackblitz, porem desta vez me enviar o link de compartilhamento geral para eu entender o código?
E que o link que enviou e apenas visualização!

Attr.

**Reply by gtrabach** [reply]
Bom dia! Segue conforme solicitado: https://stackblitz.com/edit/po-ui-st8kou?file=src%2Fapp%2Fapp.component.ts

**Reply by anliben** [reply, marked as answer]
Bom Dia, @gtrabach, tudo bem?

Notei que esta utilizando duas paginas dentro uma da outra, o ideia seria utilizar apena uma delas ou o `po-page-default` ou o po`-page-dynamic-table`

Note que o `po-page-default` e faz na página exatamente o que o `po-page-dynamic-table` faz que é deixar uma lateral esquerda de 290px para o `po-menu`!

![image](https://github.com/po-ui/po-angular/assets/45929971/5f6c5443-151d-4631-bd7c-abfcc8a740e7)

**Reply by gtrabach** [reply]
Bom dia. Sim, o problema é justamente por utilizar uma página dentro da outra. 

Existe alguma tag ou classe que posso utilizar de forma que seja ignorado os 290px do po-menu? Dessa forma, usarei essa tag/classe na po-page-dynamic-table.

**Necessidade atual:** a página que estou construindo terá outros componentes além do po-page-dynamic-table, todos os outros componentes são alinhados corretamente ao po-menu, menos o po-page-dynamic-table justamente por ele criar este espaço na lateral.

**Reply by anliben** [reply]
Boa Tarde. Podes criar uma validação para isso no seu projeto, o `po-page-default` ira sempre criar esta forma!

Podes personalizar de alguma forma lembrando que o `po-page-default` utiliza a classe `.po-page` e como o `po-page-dynamic-table` também e uma página ira dá conflito nas classes!

Avalie uma forma diferente, acredito, pois não ha uma tag para seu caso em específico.

Attr.
